### PR TITLE
Big validator rotation tests

### PR DESCRIPTION
**Pull request by timorl**

### CODE REVIEW

File: fabfile.py [hunk 1]
```diff
@@ -272,6 +272,46 @@ def upgrade_binary(conn):
     conn.run(f'dtach -n `mktemp -u /tmp/dtach.XXXX` sh /home/ubuntu/cmd.sh')
 
 
+# ======================================================================================
+#                                      validator rotation
+# ======================================================================================
+
+@task
+def send_cli_binary(conn):
```
Comment (kostekIV) [inline fabfile.py:280]: u can use `send_zip` here

File: shell.py [hunk 2]
```diff
@@ -149,6 +149,21 @@ def instances_state_in_region(region_name=default_region(), tag='dev'):
...
     return states
 
+def get_pid_ips(pids):
+    if pids is None:
+        with open('addresses', 'r') as f:
+            return map(lambda line: line.strip(), f.readlines())
+    else:
+        ips = []
+        for pid in pids:
+            line = int(pid)+1
```
Comment (kostekIV) [inline shell.py:159]:
```suggestion
            line = int(pid) + 1
```

**Review by kostekIV — APPROVED** [review]

**Review by Yithis — APPROVED** [review]

File: fabfile.py [hunk 3]
```diff
@@ -171,7 +171,7 @@ def create_dispatch_cmd(conn, pid):
     ]
     debug_flags = [
         '-lafa=debug',
```
Comment (Yithis) [inline fabfile.py:173]: It doesn't exist anymore

File: fabfile.py (hunk 1 above)
Comment (timorl) [reply fabfile.py:280]: Done, also made the `send_binary` function do the same, and fixed the order of arguments in `send_zip` (the only usages had the zip file and binary in the wrong order?).

File: fabfile.py (hunk 3 above)
Comment (timorl) [reply fabfile.py:173]: Removed.

File: shell.py (hunk 2 above)
Comment (timorl) [reply shell.py]: Done.

File: fabfile.py (hunk 1 above)
Comment (kostekIV) [reply fabfile.py:275]: lul

**Pull request merged**